<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardPage</class>
 <widget class="QWizardPage" name="WizardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="xlab_layout">
     <item>
      <widget class="QCheckBox" name="xlabCheckBox">
       <property name="text">
        <string>set xlabel?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>xlabel:</string>
       </property>
       <property name="buddy">
        <cstring>xlab_le</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="xlab_le">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="ylab_layout">
     <item>
      <widget class="QCheckBox" name="ylabCheckBox">
       <property name="text">
        <string>set ylabel?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>ylabel:</string>
       </property>
       <property name="buddy">
        <cstring>ylab_le</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ylab_le">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="xlim_layout">
     <item>
      <widget class="QCheckBox" name="xlimCheckBox">
       <property name="toolTip">
        <string>x axis limits. If not checked, the function tries to set the xaxis limits to some sensible values.</string>
       </property>
       <property name="text">
        <string>set xlims?</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>[</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="xlimLowSpinBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <double>-10000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>,</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="xlimHighSpinBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <double>-10000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>]</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>38</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="ylim_layout_2">
     <item>
      <widget class="QCheckBox" name="ylimCheckBox">
       <property name="toolTip">
        <string>y axis limits. If not checked, the function tries to set the yaxis limits to some sensible values.</string>
       </property>
       <property name="text">
        <string>set ylims?</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="ylim_layout">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>[</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="ylimLowSpinBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <double>-10000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>,</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="ylimHighSpinBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <double>-10000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>]</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>38</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="label25">
       <property name="text">
        <string>status: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="status_lbl">
       <property name="text">
        <string>actualstatus</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
